scripts/isct_test_function.py

```diff
@@ -433,6 +433,39 @@ def get_parser():
         print results_display.to_string()
         print 'Status: 0: Passed | 1: Crashed | 99: Failed | 200: Input file(s) missing | 201: Ground-truth file(s) missing'
 
+        if verbose == '2':
```

Comment:
I think we decided to always use verbose as int instead of str (converted when retrieving the parameter). @peristeri any thought?